relay-server/src/actors/events.rs

```diff
@@ -311,20 +316,13 @@ impl EventProcessor {
             let mut session = match SessionUpdate::parse(&payload) {
                 Ok(session) => session,
                 Err(error) => {
-                    return sentry::with_scope(
-                        |scope| {
-                            scope.set_tag("project", project_id);
-                            if let Some(ref client) = client {
-                                scope.set_tag("sdk", client);
-                            }
-                            scope.set_extra("session", String::from_utf8_lossy(&payload).into());
-                        },
-                        || {
-                            // Skip gracefully here to allow sending other sessions.
-                            log::error!("failed to store session: {}", LogError(&error));
-                            false
-                        },
-                    );
+                    sentry::configure_scope(|scope| {
+                        scope.set_extra("session", String::from_utf8_lossy(&payload).into());
```

Comment:
`configure_scope` is simpler to use here, but it will leak into errors that happen outside of this context. We probably don't want to log the session payload on user report errors, etc.